editor/plugins/lightmap_gi_editor_plugin.cpp

```diff
@@ -174,7 +174,7 @@ LightmapGIEditorPlugin::LightmapGIEditorPlugin() {
 	bake->set_icon(EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("Bake"), EditorStringName(EditorIcons)));
 	bake->set_text(TTR("Bake Lightmaps"));
 	bake->hide();
-	bake->connect("pressed", Callable(this, "_bake"));
+	bake->connect("pressed", callable_mp(this, &LightmapGIEditorPlugin::_bake));
```

Comment:
If this `connect()` is the only reason that `"_bake"` is bound in `_bind_methods()`, then that could be removed too.

But I'm assuming this change isn't actually required, right? While using `callable_mp()` is certainly more idiomatic, this is unrelated to the main focus of this PR.

Reply:
Right

This was a last minute change before submitting the PR. Errors were printing durring editor startup about not being able to connect `"pressed"` to a null object. See my comment on [`Object::connect` line 1306](https://github.com/godotengine/godot/pull/82695/files#r1343282910). I suspect there may be many of these, I only just noticed the two here as they triggered errors on startup.

Reply:
I am concerned about these classes not being registered property, but I think that's for a different PR. This PR as it is now should fine for any such classes.